packages/signers/src/__tests__/add-signers-test.ts

```diff
@@ -190,6 +191,43 @@ describe('addSignersToTransactionMessage', () => {
         ]);
     });
 
+    it('add updates the fee payer if a matching signer is provided', () => {
```

Comment:
Nit: remove the `add` (also line 212)

```suggestion
    it('updates the fee payer if a matching signer is provided', () => {
```